addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -283,21 +283,23 @@ class ImLivechatChannelRule(models.Model):
     regex_url = fields.Char('URL Regex',
         help="Regular expression specifying the web pages this rule will be applied on.")
     action = fields.Selection([
-        ('display_button', 'Display the LiveChat Button'),
-        ('auto_popup', 'Autopop LiveChat'),
-        ('hide_button', 'Hide all buttons')], string='Action', required=True, default='display_button',
-        help="* 'Display the button' displays the chat button on the pages.\n"\
-             "* 'Auto popup' displays the button and automatically open the conversation pane.\n"\
-             "* 'Hide the button' hides the chat button on the pages.\n")
-    auto_popup_timer = fields.Integer('Auto popup timer', default=0,
-        help="Delay (in seconds) to automatically open the conversation window. Note: the selected action must be 'Auto popup' otherwise this parameter will not be taken into account.")
+        ('display_button', 'Show'),
+        ('display_button_and_text', 'Show with notification'),
+        ('auto_popup', 'Open automatically'),
+        ('hide_button', 'Hide')], string='Live Chat Button', required=True, default='display_button',
+        help="* 'Show' displays the chat button on the pages.\n"\
+             "* 'Show with notification' is 'Show' in addition to a floating text just next to the button.\n"\
+             "* 'Open automatically' displays the button and automatically open the conversation pane.\n"\
+             "* 'Hide' hides the chat button on the pages.\n")
+    auto_popup_timer = fields.Integer('Open automatically timer', default=0,
+        help="Delay (in seconds) to automatically open the conversation window. Note: the selected action must be 'Open automatically' otherwise this parameter will not be taken into account.")
```

Comment:
do you know if this kind of change requires migration scripts?

Reply:
I'm pretty sure changing `help` do not need migration scripts